test/utils/server.js

```diff
@@ -90,7 +90,7 @@ app
   .get('/builds.html', sendFile(root + '/test/browser_build_unit_tests.html'))
 
   // support
-  .get('/expect.js', sendFile(root + '/node_modules/expect.js/expect.js'))
+  .get('/expect.js', sendFile(root + '/node_modules/expect.js/index.js'))
```

Comment:
this was bizarre... `expect.js/` doesn't have an `expect.js` file in it, only `index.js` but I sense that one day it had been different.